packages/types/src/transport.ts

```diff
@@ -4,6 +4,15 @@ import { Response } from './response';
 import { SdkMetadata } from './sdkmetadata';
 import { Session, SessionAggregates } from './session';
 
+export enum Outcome {
+  BeforeSend = 'before_send',
+  EventProcessor = 'event_processor',
+  NetworkError = 'network_error',
+  QueueSize = 'queue_size',
```

Comment:
I find `queue_size` hard to understand as an "outcome". Perhaps if we described what "outcome" means I'd have a easier time thinking about it, but as is I think of it as description of cases that lead to an event to be discarded, and the reason here is "queue full" or "queue overflow" or "buffer full/overflow".

Reply:
Lets keep this type of conversations in the design doc to keep everyone in sync

Reply:
It's `queue_overflow` in the design doc now.